Sources/generate-swift-syntax-builder/Templates/BuildableBaseProtocolsFile.swift

```diff
@@ -34,7 +34,7 @@ let buildableBaseProtocolsFile = SourceFile {
     let listConformances: [String] = isSyntax ? [] : [syntaxType.listBuildableBaseName]
 
     ProtocolDecl(
-      modifiers: [TokenSyntax.public],
+      modifiers: [Token.public],
```

Comment:
Just a random idea, what would it take to make it possible to write implicit member expressions here, i.e. `[.public]`? Could we declare tokens that are valid modifiers as static constants on `DeclModifier`? (Though that probably wouldn't work through the layer of `ExpressibleAs...` protocols)

Reply:
Yes, I think that’s the problem, we can’t use implicit member lookup on protocols.